### PR TITLE
Build a 16.04 image with latest itzo-launcher

### PR DESCRIPTION
**Pull request by ldx**
I reverted back to 16.04 for now, adding podman from the 18.04 kubic repo. It installs and works fine based on my (limited) testing; the only gotcha was that its systemd unit file used a parameter that is not supported by systemd on 16.04.

### CODE REVIEW

File: .github/workflows/build.yml [hunk 1]
```diff
@@ -1,7 +1,7 @@
 on:
   push:
     branches:
-      - master
+      - "*"
```
Comment (hidalgopl) [inline .github/workflows/build.yml:4]: we probably want to revert this change

**Review by hidalgopl — APPROVED** [review]
nice one!

File: .github/workflows/build.yml (hunk 1 above)
Comment (ldx) [reply .github/workflows/build.yml:4]: I don't mind building a new image for every branch, so people can test their changes, WDYT?

I'll create an issue to do a periodic cleanup of old dev images.

Comment (hidalgopl) [reply .github/workflows/build.yml:4]: Ah, ok. Yeah, that may be useful. So it can be even simplified to `on: push`
